tf/aks/outputs.tf

```diff
@@ -0,0 +1,47 @@
+/**
+ * Outputs for AKS Module
+ *
+ * This Terraform file defines the outputs for the AKS (Azure Kubernetes Service) module.
+ * These outputs provide information about the created AKS cluster and its resources.
+ */
+
+output "resource_group_name" {
+  value       = module.aks.resource_group_name
+  description = "The name of the resource group where the AKS cluster is deployed."
+}
+
+output "aks_client_key" {
+  value       = module.aks.aks_client_key
+  description = "The client key used for authenticating with the AKS cluster."
+}
+
+output "aks_client_certificate" {
+  value       = module.aks.aks_client_certificate
+  description = "The client certificate used for authenticating with the AKS cluster."
+}
```

Comment:
This a sensitive output that has to be redacted from the Terraform logs using: sensitive   = true
https://developer.hashicorp.com/terraform/tutorials/configuration-language/outputs#redact-sensitive-outputs